src/encoded/static/components/matrix.js

```diff
@@ -449,7 +449,16 @@ class MatrixPresentation extends React.Component {
                 return { expandedRowCategories: prevState.expandedRowCategories.concat(category) };
             }
 
-            // Category does exist in array, so remove it.
+            // Category does exist in array
+            // Move close to header
+            const header = document.querySelector(`#${category.replace(/\s/g, '_')}`);
+            header.scrollIntoView({
+                behavior: 'smooth',
+                block: 'end',
+                inline: 'nearest',
```

Comment:
Is `inline: 'nearest'` important? It doesn't seem like there is any horizontal scrolling here.

Reply:
No, also, "inline" is set to "nearest" by default. I removed it.